### PR TITLE
Add inventory group for DLS team

### PR DESCRIPTION
**Pull request by acozine**
To facilitate tasks like adding infrastructure to CheckMK, we would like to track inventory groups for each dev team. This PR adds a group for DLS. The initial list is based on the applications listed in the [DLS handbook](https://github.com/pulibrary/dls-handbook/blob/main/applications.md). 

Also adds a variable for the CheckMK folder to all three teams' groups. Eventually this will support running the `checkmk_agent` playbook against a wider variety of infrastructure at once, rather than passing `-e checkmk_folder=<folder_name>`.

### CODE REVIEW

File: inventory/by_team/dls [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[dls:children]
+abid_production
+abid_staging
+dpul_production
+dpul_staging
+figgy_production
+figgy_staging
+# imagecat inventory should be added here
```
Comment (hackartisan) [inline inventory/by_team/dls:8]: imagecat is running on the nomad cluster

**Review by hackartisan — COMMENTED** [review]
looks good!

File: inventory/by_team/dls (hunk 1 above)
Comment (acozine) [reply inventory/by_team/dls]: Addressed in 801d805cbc5c84070c19b264d1ec5e58de06871d.

**Comment by acozine** [comment]
This is a different approach to group variables than we have traditionally taken in this repo. I'm wondering how folks view this approach?

File: inventory/by_team/dls [hunk 2]
```diff
@@ -5,10 +5,10 @@ dpul_production
 dpul_staging
 figgy_production
 figgy_staging
-# imagecat inventory should be added here
 lae_production
 lae_staging
 nomad_cluster
+# nomad cluster runs the dpul-c and imagecat projects
```
Comment (hackartisan) [inline inventory/by_team/dls:11]: 👍🏻

**Comment by hackartisan** [comment]
> This is a different approach to group variables than we have traditionally taken in this repo. I'm wondering how folks view this approach?

It is not what I would have expected, I didn't know you could put vars in inventory files.

What would be the alternative here? Putting this value in each application's group_vars?

**Review by bess — APPROVED** [review]

**Comment by bess** [comment]
I haven't seen this pattern before, but it looks fine to me. Thanks for putting so much thought into how to organize monitoring sustainably.

File: inventory/by_team/dls [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+nomad_clients_workers
+pulfalight_production
+pulfalight_production_workers
+pulfalight_qa
+pulfalight_qa_workers
+pulfalight_staging
+pulmap_production
+pulmap_staging
+[dls:vars]
+checkmk_folder=linux/dls
```
Comment (maxkadel) [inline inventory/by_team/dls:21]: Newline at end of file - 

```suggestion
checkmk_folder=linux/dls

```

**Pull request merged**